SCClassLibrary/JITLib/ProxySpace/NodeProxy.sc

```diff
@@ -3,24 +3,39 @@ NodeProxy : BusPlug {
 	var <group, <objects, <nodeMap, <children;
 	var <loaded=false, <>awake=true, <paused=false;
 	var <>clock, <>quant;
-	classvar <>buildProxyControl, <>buildProxy, <defaultFadeTime=0.02;
+	var <blockSize, <upsample;
 
+	classvar <>buildProxyControl, <>buildProxy, <defaultFadeTime=0.02;
 
-	*new { | server, rate, numChannels, inputs |
-		var res = super.new(server).init;
+	*new { | server, rate, numChannels, inputs, blockSize, upsample |
+		var res = super.new(server).init(blockSize, upsample);
 		if(rate.notNil or: { numChannels.notNil }) { res.initBus(rate, numChannels) };
 		inputs.do { |o| res.add(o) };
 		^res
 	}
 
+	blockSize_ { |inBlockSize|
```

Comment:
We may want this change in setting to resend any running synths? (counts for everything below too)

Reply:
yes, was considering that: 
As now, it is more of a default setting you prepare in advance, which gets used 
next time you do something, and where lower levels override higher level defaults, 
so things can get more differentiated down the hierarchy. 
Dont know if this is the best option - there could be an elastic-like mode reblocking, 
or an update flag in the setter.